tests/Unit/ParallelAlgorithms/LinearSolver/ConjugateGradient/CMakeLists.txt

```diff
@@ -20,5 +20,8 @@ add_dependencies(
   module_GlobalCache
   )
 
-add_linear_solver_algorithm_test("ConjugateGradientAlgorithm")
-add_linear_solver_algorithm_test("DistributedConjugateGradientAlgorithm")
+# Temporarily comment out until the elliptic solvers is updated to properly
+# register with observers. Currently an observation ID is used multiple times
+# by different events/components.
+# add_linear_solver_algorithm_test("ConjugateGradientAlgorithm")
```

Comment:
Please apply this patch, then you can re-enable these tests:

```patch
diff --git a/src/ParallelAlgorithms/LinearSolver/ConjugateGradient/ConjugateGradient.hpp b/src/ParallelAlgorithms/LinearSolver/ConjugateGradient/ConjugateGradient.hpp
index 04dda3d84..9ebaaf096 100644
--- a/src/ParallelAlgorithms/LinearSolver/ConjugateGradient/ConjugateGradient.hpp
+++ b/src/ParallelAlgorithms/LinearSolver/ConjugateGradient/ConjugateGradient.hpp
@@ -112,8 +112,7 @@ struct ConjugateGradient {
    */
   using initialize_element = detail::InitializeElement<FieldsTag, OptionsGroup>;
 
-  using register_element =
-      observers::Actions::RegisterWithObservers<observe_detail::Registration>;
+  using register_element = tmpl::list<>;
 
   /*!
    * \brief Reset the linear solver to its initial state.
diff --git a/src/ParallelAlgorithms/LinearSolver/Gmres/Gmres.hpp b/src/ParallelAlgorithms/LinearSolver/Gmres/Gmres.hpp
index 56c985167..59bd5f08e 100644
--- a/src/ParallelAlgorithms/LinearSolver/Gmres/Gmres.hpp
+++ b/src/ParallelAlgorithms/LinearSolver/Gmres/Gmres.hpp
@@ -147,8 +147,7 @@ struct Gmres {
   using initialize_element =
       detail::InitializeElement<FieldsTag, OptionsGroup, Preconditioned>;
 
-  using register_element =
-      observers::Actions::RegisterWithObservers<observe_detail::Registration>;
+  using register_element = tmpl::list<>;
 
   /*!
    * \brief Reset the linear solver to its initial state.
```

I can deal with further refinements to how the parallel linear solvers do observations in follow-up PRs.